realsense2_camera/test/live_camera/rosci.py

```diff
@@ -0,0 +1,144 @@
+# Copyright 2023 Intel Corporation. All Rights Reserved.
```

Comment:
2024

Reply:
Corrected in commit #3

Reply:
GHA failing for 2024, hence reverted to 2023 again w/ commit # 4

Reply:
@Arun-Prasad-V  / @SamerKhshiboun haven't we fixed that to be generic?